zitadel/helper/helper.go

```diff
@@ -124,8 +124,19 @@ func GetStringFromAttr(ctx context.Context, attrs map[string]attr.Value, key str
 func DescriptionEnumValuesList(enum map[int32]string) string {
 	str := ", supported values: "
 	values := make([]string, len(enum))
-	for i := 0; i < len(enum); i++ {
-		values[i] = enum[int32(i)]
+	highest := 0
```

Comment:
I assume you are trying to sort them? Why does it not already sort correctly? We access the values by the indexed key, right?

Reply:
No the problem is that ENUMs can technically start with wherever and also numbers can be left out, that's why we first have to find out which is the highest number and then go from 0 to the highest to find out which are existing.

Reply:
ok, got it